handlers/product-move-api/src/main/scala/com/gu/productmove/endpoint/available/AvailableProductMovesEndpoint.scala

```diff
@@ -44,44 +53,121 @@ object AvailableProductMovesEndpoint {
       .serverLogic[TIO] { subscriptionName => run(subscriptionName).tapEither(result => ZIO.log("result tapped: " + result)).map(Right.apply) }
   }
 
+  // sub to test on: "A-S00334930"
   private def run(subscriptionName: String): TIO[OutputBody] =
     runWithEnvironment(subscriptionName).provide(
       AwsS3Live.layer,
       AwsCredentialsLive.layer,
       SttpClientLive.layer,
       ZuoraClientLive.layer,
       GetSubscriptionLive.layer,
+      GetCatalogueLive.layer,
+      GetAccountLive.layer,
       ZuoraGetLive.layer,
       GuStageLive.layer,
     )
 
-  private[productmove] def runWithEnvironment(subscriptionName: String): ZIO[GetSubscription, String, OutputBody] =
+  private val freeTrialDays = 14
+
+
+  def getAvailableSwitchRatePlans(zuoraProductCatalogue: ZuoraProductCatalogue, ratePlanNames: List[String]): List[ZuoraProductRatePlan] =
+    val productRatePlans = for {
+      product <- zuoraProductCatalogue.products.filter(ratePlanNames contains _.name)
```

Comment:
I'm not against bracket free full stop, but generally I include the brackets etc for clarity in this situation
```suggestion
      product <- zuoraProductCatalogue.products.filter(ratePlanNames.contains(_.name))
```

Reply:
just wanted to check if this comment was missed or you disagree?
```suggestion
      product <- zuoraProductCatalogue.products.filter(ratePlanNames.contains(_.name))
```

Reply:
I actually got a compiler error when changing it to the suggested version:
<img width="821" alt="Screenshot 2022-08-24 at 14 53 23" src="https://user-images.githubusercontent.com/91546670/186436436-b7b0f9ad-ff0a-4b68-91ad-4b968b089bf4.png">

Reply:
my mistake, I didn't realise scala bracketed the underscores that way, try this instead
```suggestion
      product <- zuoraProductCatalogue.products.filter(product => ratePlanNames.contains(product.name))
```